strongback/docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Houston docker file
+# Builds an ubuntu base with liftoff for packaging debian applications
```

Comment:
Alright we can change this down the line if it causes us trouble, right now it's working fine, although there probably is a slight performance drop here.